File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.cs
using System;

public class Solution 
{
    public int[] solution(int[] num_list) 
    {
        int[] evenoddList = new int[2];
        for(int i=0; i<num_list.Length; i++)
        {
            if(num_list[i]%2 == 0)
            {
                evenoddList[0]++;
            }
            else
            {
                evenoddList[1]++;
            }
        }
        return evenoddList;
    }
}

File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> num_list) 
{
    vector<int> even_odd_vec(2,0);
    for(int &i : num_list)
    {
        if(i%2 == 0) {even_odd_vec[0]++;}
        else if(i%2 == 1) {even_odd_vec[1]++;}
    }
    return even_odd_vec;
}